pom.xml

```diff
@@ -10,16 +10,29 @@
   <!-- comment -->
   <!-- comment -->
 
+  <properties>
+    <jackson-version>2.10.0</jackson-version>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.apache.struts</groupId>
+        <artifactId>struts2-core</artifactId>
+        <version>2.3.3</version>
```

Comment:
### :thinking: Nexus IQ found policy violations introduced by:

<details open>
  <summary title="Threat Level: 10 of 10"><img alt="T10" src="https://cdn.sonatype.com/iq-for-scm/1.0/red-bar.png">
    <b>10&nbsp;&nbsp;&nbsp; org.apache.struts : struts2-core : 2.3.3</b></summary>
<p></p>

:shield: **Bumping to version 2.5.22** will resolve these violations (as of _Nov 20, 2020_)

Threat (of 10) | Policy | Violation Details
--- | --- | --- |
10 | Security-Critical | <b>Critical risk CVSS score:</b><ul><li>Found security vulnerabilities: [CVE-2013-1966](http://localhost:8070/ui/links/vln/CVE-2013-1966), [CVE-2013-2134](http://localhost:8070/ui/links/vln/CVE-2013-2134), [CVE-2013-2135](http://localhost:8070/ui/links/vln/CVE-2013-2135), [CVE-2013-2251](http://localhost:8070/ui/links/vln/CVE-2013-2251), [CVE-2013-4316](http://localhost:8070/ui/links/vln/CVE-2013-4316), [CVE-2016-3082](http://localhost:8070/ui/links/vln/CVE-2016-3082), [CVE-2016-4436](http://localhost:8070/ui/links/vln/CVE-2016-4436), [CVE-2017-12611](http://localhost:8070/ui/links/vln/CVE-2017-12611)</li></ul>
9 | Security-High | <b>High risk CVSS score:</b><ul><li>Found security vulnerabilities: [CVE-2013-2115](http://localhost:8070/ui/links/vln/CVE-2013-2115), [CVE-2014-0113](http://localhost:8070/ui/links/vln/CVE-2014-0113), [CVE-2015-5209](http://localhost:8070/ui/links/vln/CVE-2015-5209), [CVE-2016-0785](http://localhost:8070/ui/links/vln/CVE-2016-0785), [CVE-2016-4430](http://localhost:8070/ui/links/vln/CVE-2016-4430), [CVE-2018-11776](http://localhost:8070/ui/links/vln/CVE-2018-11776)</li></ul>
7 | Security-Medium | <b>Medium risk CVSS score:</b><ul><li>Found security vulnerabilities: [CVE-2012-4386](http://localhost:8070/ui/links/vln/CVE-2012-4386), [CVE-2013-2248](http://localhost:8070/ui/links/vln/CVE-2013-2248), [CVE-2013-4310](http://localhost:8070/ui/links/vln/CVE-2013-4310), [CVE-2014-0094](http://localhost:8070/ui/links/vln/CVE-2014-0094), [CVE-2014-7809](http://localhost:8070/ui/links/vln/CVE-2014-7809), [CVE-2015-5169](http://localhost:8070/ui/links/vln/CVE-2015-5169), [CVE-2016-4003](http://localhost:8070/ui/links/vln/CVE-2016-4003)</li></ul>

</details>